src/System.Runtime.Extensions/tests/System/IO/Path.Combine.cs

```diff
@@ -5,279 +5,276 @@
 using System.IO;
 using Xunit;
 
-namespace PathTests
+public class PathCombineTests
 {
-    public class CombineTests
+    private static String s_separator = "\\"; // Was Path.DirectorySeparatorChar
```

Comment:
This makes the combine tests platform-dependent now.  Why not use Path.DirectorySeparatorChar?